### PR TITLE
Improve website API reference

### PR DESCRIPTION
**Pull request by VedantS73**
The website I accessed was not having enough documentation and I personally found it difficult to find references on how to use controlled and custom rows/ columns.
So I added it by myself and created this request. I would love to contribute more to this project.

### CODE REVIEW

File: website/docs/usage.md [hunk 1]
```diff
@@ -25,6 +25,40 @@ const App = () => {
 };
 ```
 
+## Custom Columns and Rows
+
+The Spreadsheet component requires the `columnLabels` prop or the `rowLabels` prop which accepts array. If no `columnLabels` is passed, it defaults to alphabetical labels and if no `rowLabels` are passed it defaults to row index labels.
```
Comment (iddan) [inline website/docs/usage.md:30]: I think requires is not the right wording as these are optional parameters. How about "accepts" instead?

File: website/docs/usage.md [hunk 2]
```diff
@@ -34,9 +68,28 @@ import Spreadsheet from "react-spreadsheet";
...
+        t[i][2] = {value:"🍪"}
+      }
+      else if(string.value === "Pizza"){
+        t[i][2] = {value:"🍕"}
+      }
+      else {
+        t[i][2] = {value:"❓"}
+      }
+      setData(t)
+    }
```
Comment (iddan) [inline website/docs/usage.md:92]: In react it's recommended not to mutate source data. Make sure to duplicate the array before mutating it.

Comment (VedantS73) [reply website/docs/usage.md:92]: okay I'll make the required changes.

**Comment by VedantS73** [comment]
@iddan I have made the changes. Kindly note and let me know if anything more needed.

**Review by iddan — APPROVED** [review]

**Comment by iddan** [comment]
Good stuff. Thank you!

**Comment by VedantS73** [comment]
Hey I have no idea why this check was not successful? can you please help.

**Comment by VedantS73** [comment]
Hey @iddan ran prettier on usage.md, can you please initiate the merge again?

**Comment by iddan** [comment]
It's up now!

**Pull request merged**